notebooks/random_one_point_prediction.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import torch
+import cv2
+
+import sys
+sys.path.append("..")
+from segment_anything import sam_model_registry, SamAutomaticMaskGenerator, SamPredictor
+
+
+# numpy version
+def pixAcc(predicted, target):
+    same = (predicted == target).sum()
+    w, h = target.shape
+    return same / (w * h)
+
+# input: bool matrix
+def IOU(predicted , target):
+    intersection = np.logical_and(target, predicted).sum()
+    union = np.logical_or(target, predicted).sum()
+    if union == 0:
+        iou_score = 0
+    else :
+        iou_score = intersection / union
+    return iou_score
+def gt_to_anns(mask_gt):
+    labels = np.unique(mask_gt)
+    anns = []
+    for label in labels:
+        # skip background
+        if label == 0:
+            continue
+        mask = np.all(mask_gt == label, axis=-1)
+        # 1 ramdon point from mask
+        num_point = 1
+        indices = np.argwhere(mask)
+        # swap x y
+        indices[:,[1,0]] = indices[:,[0,1]]
```

Comment:
This is because `np.argwhere` returns `i, j` indicies (which are effectively `y, x`) but SAM expects `x, y`?

Reply:
Yes, I will leave comment in file later